utils.ts

```diff
@@ -572,6 +572,28 @@ export async function applyPackageOverrides(
 			...pkg.pnpm.overrides,
 			...overrides,
 		}
+		// check `overrides` in pnpm-workspace.yaml
+		const pnpmWorkspaceFile = path.join(dir, 'pnpm-workspace.yaml')
+		if (fs.existsSync(pnpmWorkspaceFile)) {
+			const pnpmWorkspaceContent = await fs.promises.readFile(
+				pnpmWorkspaceFile,
+				'utf-8',
+			)
+			if (/^overrides:/m.test(pnpmWorkspaceContent)) {
+				delete pkg.pnpm.overrides
```

Comment:
Is this line needed?

Reply:
Yes, when `overrides` exist both in `package.json` and `pnpm-workspace.yaml`, `pnpm-workspace.yaml`'s `overrides` are ignored.

Reply:
Ah, it is for `await fs.promises.writeFile(pkgFile, JSON.stringify(pkg, null, 2), 'utf-8')`.